### PR TITLE
[test] make a constant_struct_with_padding work when cross-compiling.

### PR DESCRIPTION
**Pull request by huonw**
Addresses rdar://problem/29173452.

### CODE REVIEW

File: test/IRGen/constant_struct_with_padding.sil [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// RUN: %swift -module-name main %s -emit-ir -o - | %FileCheck %s
+// RUN: %target-swift-frontend -c -module-name main %s -emit-ir -o - | %FileCheck %s
```
Comment (jrose-apple) [inline test/IRGen/constant_struct_with_padding.sil]: `-c` and `-emit-ir` are in the same family of actions, so `-c` is unnecessary.

**Comment by huonw** [comment]
@swift-ci Please test

**Comment by huonw** [comment]
@swift-ci Please smoke test and merge

**Comment by huonw** [comment]
@swift-ci Please smoke test and merge

**Comment by huonw** [comment]
@swift-ci Please smoke test and merge

**Pull request merged**